backend/lib/endpoints/organisations.js

```diff
@@ -16,6 +17,206 @@ async function routes(app) {
   const Comment = app.mongo.model("Comment");
   const Organisation = app.mongo.model("OrganisationUser");
   const Post = app.mongo.model("Post");
+  const User = app.mongo.model("IndividualUser");
+
+  const ORGS_PAGE_SIZE = 10;
+
+  app.get(
+    "/search",
+    {
+      preValidation: [app.authenticateOptional],
+      schema: searchOrganisationsSchema,
+    },
+    async (req) => {
+      const { query, userId } = req;
+      const {
+        ignoreUserLocation,
+        filter,
+        keywords,
+        limit,
+        objective,
+        skip,
+        includeMeta,
+      } = query;
+      const queryFilters = filter ? JSON.parse(decodeURIComponent(filter)) : {};
+      let user;
+      let userErr;
+      if (userId) {
+        [userErr, user] = await app.to(User.findById(userId));
+        if (userErr) {
+          req.log.error(userErr, "Failed retrieving user");
+          throw app.httpErrors.internalServerError();
+        } else if (user === null) {
+          req.log.error(userErr, "User does not exist");
+          throw app.httpErrors.forbidden();
+        }
+      }
+
+      /* eslint-disable sort-keys */
+      const filters = [{ type: { $ne: "Individual" } }];
+
+      // un-comment if user shouldnt find their owned organizations
```

Comment:
Do we need this comment? I think it will just be confusing for someone else reading the code later.